nvflare/fuel/f3/drivers/aio_grpc_driver.py

```diff
@@ -328,6 +328,9 @@ async def _start_connect(self, connector: ConnectorInfo, aio_ctx: AioContext, co
             msg_iter = stub.Stream(connection.generate_output())
             conn_ctx.conn = connection
             await connection.read_loop(msg_iter)
+        except asyncio.CancelledError:
```

Comment:
We have a few other `except Exception` without this. Please add this catch clause to all of them.